src/components/feedback/Toast/usePromiseToast.ts

```diff
@@ -10,13 +17,21 @@ const usePromiseToast = () => {
      * @param {(String|Object)} pending The message to be shown while promise in pending or the entire object with all the configurations
      * @param {(String|Object)} success The message to be shown when promise completed successfully or the entire object with all the configurations
      * @param {(String|Object)} error The message to be shown when promise was rejected or the entire object with all the configurations
+     * @param {ToastOptions} options Additional options passed to the toast
      */
-    (promise: Promise<unknown>, pending: string | object, success: string | object, error: string | object) => {
+    (promise: Promise<unknown>, pending: string | object, success: string | object, error: string | object, { transitionType, ...restOptions } = {} as ToastOptions) => {
```

Comment:
Here, in place of "as"... I thik there should be ":". To enforce the type.. not to cast the type you are receiving.

Reply:
won't do